js/load.js

```diff
@@ -0,0 +1,246 @@
+const worksWrapper = document.querySelector('#works .wrapper');
+
+const work = {
+  id: 1,
+  name: 'Tonic',
+  description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+  fullDescription: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  LongDescription: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+  image: {
+    src: `./images/snapshoot/snapshoot_${this.id}.svg`,
+    alt: 'Project image',
+  },
+  info: {
+    author: 'RAMOUN',
+    type: 'Back End Dev',
+    year: '2015',
+  },
+  technologies: ['html', 'css', 'JavaScript', 'cpp'],
+  buttons: {
+    live: {
+      text: 'see live',
+      link: '#',
+      imgSrc: './images/arrow.svg',
+    },
+    source: {
+      text: 'see source',
+      link: '#',
```

Comment:
- [ ] For this milestone, you are required to have all your projects in an array of objects. Each object should contain details of a project.